packages/api/src/index.ts

```diff
@@ -36,8 +45,13 @@ async function bootstrap() {
   // Start server
   await startStandaloneServer(server, {
     listen: { port: 4000 },
-    context: async () => {
-      return { knex };
+    context: async ({ req }) => {
+      return {
+        knex,
+        user: {
+          loginToken: req.headers.authorization,
+        },
```

Comment:
Kept this as an object in the event we want to pass more information along with the request in the future. I'm ok with consolidating it to `token` or something if it makes more sense